core/templates/core/landing_page_international.html

```diff
@@ -1,11 +1,13 @@
 {% extends 'core/base_international.html' %}
 {% load static from staticfiles %}
 {% load trans from i18n %}
+{% load banner from directory_components_tags %}
 
 {% block content %}
-{% if features.NEWS_SECTION_ON %}
-  <div class="container" dir="ltr">
-    {% include 'core/includes/eu-exit-banner.html' %}
+
+{% if features.NEWS_SECTION_ON and cms_component and page.articles_count > 0 %}
+  <div class="container banner-container" dir="{{ component_is_rtl|yesno:'rtl,ltr' }}">
```

Comment:
`component_is_rtl` is slightly misleading because arabic is not right to left: it's bidirectional because it can contain english words within it: so some of the text is RTL and some is LTR...hence django calls it "bidirectional" or "bidi".

so `component_is_bidi` is more accurate :)

Reply:
oh sure. i was thinking in terms of what `dir` attribute the html component needed but this makes sense